api/src/test/java/org/openmrs/EncounterTest.java

```diff
@@ -1262,4 +1264,43 @@ public void multipleAddingAndRemovingOfSameProvider_shouldNotFail() {
 
 	}
 
+	/**
+	 * @see Encounter#hasDiagnosis(Diagnosis)
+	 */
+	@Test
+	public void hasDiagnosis_shouldCheckIfEncounterHasDiagnosis(){
```

Comment:
This should be renamed to hasDiagnosis_shouldReturnTrueIfEncounterHasDiagnosis

Reply:
Okay. Done